PowerEditor/src/Notepad_plus.cpp

```diff
@@ -725,12 +725,73 @@ LRESULT Notepad_plus::init(HWND hwnd)
 	_sha512FromFilesDlg.setHashType(hash_sha512);
 	_sha512FromTextDlg.init(_pPublicInterface->getHinst(), hwnd);
 	_sha512FromTextDlg.setHashType(hash_sha512);
-
+	
 
 	//--User Define Dialog Section--//
 	int uddStatus = nppGUI._userDefineDlgStatus;
 	UserDefineDialog *udd = _pEditView->getUserDefineDlg();
 
+	void hideColumns(int numColumns) {
+    HWND editorHandle = ::GetDlgItem(_hSelf, IDC_MAIN_EDIT);
+    int lineCount = ::SendMessage(editorHandle, SCI_GETLINECOUNT, 0, 0);
+
+    for (int line = 0; line < lineCount; ++line) {
+        int lineStart = ::SendMessage(editorHandle, SCI_POSITIONFROMLINE, line, 0);
+        ::SendMessage(editorHandle, SCI_SETTARGETSTART, lineStart, 0);
+        ::SendMessage(editorHandle, SCI_SETTARGETEND, lineStart + numColumns, 0);
+        ::SendMessage(editorHandle, SCI_REPLACETARGET, 0, reinterpret_cast<LPARAM>(""));

```

Comment:
It's rather "Remove" the column than "Hide" the column.

Reply:
and that's why I suggested a macro with the name `Delete Columns...`